FORKING.md

```diff
@@ -0,0 +1,85 @@
+
+### Mit SSH
+> Tipp: Wenn du nicht weißt, was SSH-Keys sind, gehe zum Unterpunkt "mit HTTPS". Begib dich direkt dorthin, gehe nicht über Los, ziehe keine 4000 DM ein.
+
+Kopiere die Zeile, die im Textfeld bei "Clone with SSH" steht. Öffne ein Terminal, gehe in das Verzeichnis in dem ein Unterordner für die Fachschaftsseite erstellt werden soll. Tippe dann im Terminal: `git clone <Strg-Shift-V>`
+Es sollte das hier entstehen:
+```bash
+git clone git@github.com:deinusername/cyberauftritt.git
+```
+Führe den Befehl aus. Der Ordner "cyberauftritt" entsteht.
+### Mit HTTPS
+Klicke auf "Use HTTPS". Kopiere die Zeile, die im Textfeld bei "Clone with HTTPS" steht. Öffne ein Terminal, gehe in das Verzeichnis in dem ein Unterordner für die Fachschaftsseite erstellt werden soll. Tippe dann im Terminal: `git clone <Strg-Shift-V>`
+Es sollte das hier entstehen:
+```bash
+git clone git@github.com:deinusername/cyberauftritt.git
+```
+Führe den Befehl aus. Du wirst nach GitHub-Benutzernamen und dem dazugehörigen Passwort gefragt.
+Der Ordner "cyberauftritt" entsteht.
+
+## Original-Repository als Referenz hinzufügen
+
+Je nach dem wie viel auf der Seite gerade geändert wird, musst du dafür sorgen, dass deine Kopie mit dem Original auf einem Stand bleibt. Hierzu musst du einen sog. "Upstream" hinzufügen.
+- `cd /[…]/cyberauftritt`
+### Mit SSH
+> Tipp: Wenn du nicht weißt, was SSH-Keys sind, gehe zum Unterpunkt "mit HTTPS". Begib dich direkt dorthin, gehe nicht über Los, ziehe keine 4000 DM ein.
+>
+```bash
+git remote add upstream git@github.com:fsi-tue/cyberauftritt.git
+```
+### Mit HTTPS
+```bash
+git remote add upstream https://github.com/fsi-tue/cyberauftritt
+```
+Benutzernamen und Passwort eingeben.
+
+## Änderungen holen und loslegen
+> :warning: Obacht! Prüfe mit `git status`, ob du Änderungen hast, die noch nicht committed wurden. Diese Schritte gehen davon aus, dass du einen "sauberen" Fork hast, also nichts das geändert, aber noch nicht gepusht wurde.
+
+- Änderungen vom Original holen: `git fetch upstream`
+- Eigene Kopie auf gleichen Stand bringen: `git merge upstream/master`
+
+Nun ist es erst mal ein ganz normaler Commit:
+
+- Zeugs machen, das du machen willst
+- `git commit -m '<explain stuff you did>'`
+- `git push`
+
+## Änderungen zum Original überführen
+Nun musst du eine Anfrage stellen, dass die Änderungen in deiner Kopie in das Original überführt werden. GitHub nennt das einen *Pull Request*.
+Rufe deine Kopie auf GitHub auf, du siehst direkt über der Auflistung der Dateien eine Zeile wie `This branch is X commits ahead of fsi-tue:master.`. Klick in dieser Zeile auf "Pull request".
+!["Pull Request"-Button auf GitHub](https://raw.githubusercontent.com/fsi-tue/cyberauftritt/master/img/button_pr.png)
+
+Auf der erscheinden Seite sollte hoffentlich "Able to merge. These branches can be automatically merged." stehen. Wenn das nicht der Fall ist, wende dich bitte an die Admins.
```

Comment:
Vermutlich `erscheinenden` oder `erschienenen` (typo)

> Wenn das nicht der Fall ist, wende dich bitte an die Admins.

Q: Bei mir kommt da ein Fehler, was soll ich jetzt machen?
A: Git out of here!

SCNR :D
Aber wie wende ich mich überhaupt an die Admins? Evtl. fsi Mailingliste dazuschreiben und dass man einen Link zum eigenen Repo (oder zumindest den GH Benutzernamen) schicken soll? Dann könnten wir schauen was los ist und ggf. selber die Commits ohne PR holen.